config/samples/infrastructure_v1beta1_jujucluster.yaml

```diff
@@ -10,8 +10,39 @@ metadata:
   name: jujucluster-sample
 spec:
   # using loadbalancer requires metallb on vsphere
```

Comment:
This loadbalancer comment seems to have floated away from the line that it goes with

Reply:
this is  back in the correct spot now